<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPageDialog</class>
 <widget class="QDialog" name="StartPageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iRIC Start Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(250, 250, 250), stop:0.15 rgb(250,250,250), stop:0.98 rgb(235, 235, 255), stop:1 rgb(235, 235, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <pointsize>21</pointsize>
          </font>
         </property>
         <property name="text">
          <string>iRIC </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="welcomeLabel">
           <property name="text">
            <string>Welcome to iRIC!
iRIC can simulate rivers from Colorado River to the Nile.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
	border: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
	left: 5px; /* move to the right by 5px */
}

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
		stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
	border: 2px solid #C4C4C3;
	border-bottom-color: #C2C7CB; /* same as the pane color */
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	padding: 5px;
	padding-left: 5px;
	padding-right: 5px;
	min-width: 200px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #fafafa, stop: 0.4 #f4f4f4,
		stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
	border-color: #9B9B9B;
	border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
	margin-top: 3px; /* make non-selected tabs look smaller */
}

/* make use of negative margins for overlapping tabs */
QTabBar::tab:selected {
	/* expand/overlap to the left and right by 4px */
	margin-left: -4px;
	margin-right: -4px;
}

QTabBar::tab:first:selected {
	margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
	margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
	margin: 0; /* if there is only one tab, we don't want overlapping margins */
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="StartPageLaunchTabWidget" name="startTab">
      <attribute name="title">
       <string>Start Simulation Project</string>
      </attribute>
     </widget>
     <widget class="StartPageHelpTabWidget" name="helpTab">
      <attribute name="title">
       <string>Support</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="launchMiGuiButton">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #fafafa, stop: 0.4 #f4f4f4,
		stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);

}</string>
       </property>
       <property name="text">
        <string>&amp;Start iRIC-MI GUI</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StartPageLaunchTabWidget</class>
   <extends>QWidget</extends>
   <header>startpage/startpagelaunchtabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StartPageHelpTabWidget</class>
   <extends>QWidget</extends>
   <header>startpage/startpagehelptabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartPageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartPageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
